framework/src/components/storage/entities/account.js

```diff
@@ -276,6 +276,7 @@ class Account extends BaseEntity {
 
 		const defaultSort = { sort: 'balance:asc' };
 		this.extendDefaultOptions(defaultSort);
+		this.sortingFields.push('productivity');
```

Comment:
Please, add a test for this new sorting field.

Reply:
Tests are here: https://github.com/LiskHQ/lisk/blob/development/framework/test/mocha/functional/http/get/delegates.js#L456